### PR TITLE
Fix `fotmob_get_league_matches()`

### LINKED ISSUE #258: fotmob_get_league_matches has empty output
fotmob_get_league_matches(league_id = id, cached = T) where id is either 73 or other numbers has worked up until this morning, when it is now outputting an empty dataset. Any idea what is going on here?

### PR DESCRIPTION
**Pull request by tonyelhabr**
It seems that they removed/renamed the `matches.data.matchesCombinedByRound` element...

Fixes #258

### CODE REVIEW

File: R/fotmob_leagues.R [hunk 1]
```diff
@@ -303,11 +303,8 @@ fotmob_get_league_matches <- function(country, league_name, league_id, season =
     season = season
   )
   .fotmob_message_for_season(resp, season)
-  rounds <- resp$matches$data$matchesCombinedByRound
-  purrr::map_dfr(
```
Comment (tonyelhabr) [inline R/fotmob_leagues.R:307]: in addition to renaming the JSON element, it's no longer returned as a list, so there is no more need for a `map_dfr` here (the `roundName` is guaranteed to only have one type if there is just one data frame)

**Comment by JaseZiv** [comment]
@tonyelhabr The FBRef failed checks in this PR should now be sorted as there was an issue with the league metadata stored in the data repo (which I've just fixed), however there was a failed test for `fotmob_get_match_info()`...

Does this need fixing also?

**Comment by tonyelhabr** [comment]
> @tonyelhabr The FBRef failed checks in this PR should now be sorted as there was an issue with the league metadata stored in the data repo (which I've just fixed), however there was a failed test for `fotmob_get_match_info()`...
> 
> Does this need fixing also?

ah, looks like there are two additional columns returned now. this is the downside of doing a strict test for column names---it will fail when they change something on their end. on the other hand, we have transparency into their changes.

**Comment by JaseZiv** [comment]
> > @tonyelhabr The FBRef failed checks in this PR should now be sorted as there was an issue with the league metadata stored in the data repo (which I've just fixed), however there was a failed test for `fotmob_get_match_info()`...
> > Does this need fixing also?
> 
> ah, looks like there are two additional columns returned now. this is the downside of doing a strict test for column names---it will fail when they change something on their end. on the other hand, we have transparency into their changes.

Yeah absolutely. Probably the better solution too.

Thanks

**Review by JaseZiv — APPROVED** [review]

**Pull request merged**